inference-engine/ie_bridges/java/tests/InferRequestTests.java

```diff
@@ -0,0 +1,85 @@
+import junit.framework.TestCase;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import org.intel.openvino.*;
+import org.intel.openvino.InferenceEngineProfileInfo.LayerStatus;
+
+public class InferRequestTests extends TestCase {
+    ConfigTests config = new ConfigTests();
```

Comment:
Can we define a base class for all the tests with common `setUp` method? It will reduce code duplication and let to have `xmlPath` and `binPath` for test models as fields rather than methods.